test/helpers/kubectl.go

```diff
@@ -1700,7 +1737,6 @@ func (kub *Kubectl) PodsHaveCiliumIdentity(namespace, podSelector string) error
 // ready, then the number of replicas is returned.
 func (kub *Kubectl) DeploymentIsReady(namespace, deployment string) (int, error) {
 	fullName := namespace + "/" + deployment
-
```

Comment:
nit: These whitespace changes seem unnecessary.

Reply:
Sure will update the PR.